src/app/App.js

```diff
@@ -1,4 +1,24 @@
-export const App = ({options}) => {
-}
+export const App = ({ options }) => {
+  const buttonRanking = document.querySelector('.button--ranking');
 
+  function switchToHallOfFame() {
+    buttonRanking.lastElementChild.innerText = 'Rules';
+    buttonRanking.firstElementChild.classList = 'fas fa-graduation-cap';
 
+    document.querySelector('.mode__hall').hidden = false;
+    document.querySelector('.mode__rules').hidden = true;
+
+    buttonRanking.addEventListener('click', switchToRules);
+    buttonRanking.removeEventListener('click', switchToHallOfFame);
+  }
+  function switchToRules() {
+    buttonRanking.lastElementChild.innerText = 'Hall of fame';
```

Comment:
jak usuniesz ten span z html to tu po prostu będzie buttonRanking.innerText = 'Hall of fame'

Reply:
no właśnie jak nie było spana, to taka komenda buttonRanking.innerText usuwała ikonę, w sensie traktowała ją też jako innerText...